app/ui/browser-modern/reducers/pages.js

```diff
@@ -171,6 +187,18 @@ function setPageState(state, pageId, pageState) {
   });
 }
 
+function setPageUIState(state, pageId, pageUIState) {
+  return state.withMutations(mut => {
+    for (const [key, value] of Object.entries(pageUIState)) {
+      if (!(key in PageUIState.prototype)) {
+        logger.warn(`Skipping setting of \`${key}\` on page state.`);
```

Comment:
Is there an expected scenario where this would happen?  Just wondering if we should instead throw in this case to make sure setting invalid values doesn't creep into the codebase.